Update Linux install instructions (#8488)
cloudflare/cloudflare-docs: content/terraform/installing.md

## Linux

> The `terraform` binary can be installed via your distribution's package manager, eg:

Reviewer: ```suggestion
You can install the `terraform` binary via your distribution's package manager. For example:
```

```sh
$ sudo apt install terraform
```

> You can also fetch a specific version directly and place the binary in your `PATH`:

Reviewer: ```suggestion
Alternatively, you can fetch a specific version directly and place the binary in your `PATH`:
```
